### PR TITLE
(#826) Update the Encryption Passphrase images

### LINKED ISSUE #826: Update documentation for Chocolatey Central Management 0.11.0
### What You Are Seeing?
Documentation needs updating for Chocolatey Central Management 0.11.0.

### What is Expected?
All documentation is updated to reflect Chocolatey Central Management 0.11.0.

### How Did You Get This To Happen? (Steps to Reproduce)
The main documentation that will be effected lives in https://docs.chocolatey.org/en-us/central-management/

### PR DESCRIPTION
**Pull request by corbob**
## Description Of Changes

Update the Encryption Passphrase images for CCM 0.11.0

## Motivation and Context

Changes to CCM 0.11.0

## Testing

* [x] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made

* [x] Minor documentation fix (typos etc.).
* [ ] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [ ] New documentation page added.
* [ ] The change I have made should have a video added, and I have raised an issue for this.
    * Issue #

## Change Checklist

* [ ] Requires a change to menu structure (top or left hand side)/
* [ ] Menu structure has been updated

## Related Issue

Fixes #826

### CODE REVIEW

File: input/assets/images/ccm/encryption-passphrase/ccm-reset-password-link-invalid.png
Comment (st3phhays) [inline input/assets/images/ccm/encryption-passphrase/ccm-reset-password-link-invalid.png:1]: The text on this image is different. 🤔

**Review by st3phhays — CHANGES_REQUESTED** [review]

File: input/assets/images/ccm/encryption-passphrase/ccm-reset-password-link-invalid.png
Comment (corbob) [reply input/assets/images/ccm/encryption-passphrase/ccm-reset-password-link-invalid.png:1]: Oooh, I see how we got that image. I'll get this and the email validation ones redone.

File: input/assets/images/ccm/encryption-passphrase/ccm-admin-encryption-location.png
Comment (st3phhays) [inline input/assets/images/ccm/encryption-passphrase/ccm-admin-encryption-location.png:1]: This one that is deleted looks new. Was the wrong one added on accident earlier?

File: input/assets/images/ccm/encryption-passphrase/ccm-reset-password-link-invalid.png
Comment (corbob) [reply input/assets/images/ccm/encryption-passphrase/ccm-reset-password-link-invalid.png:1]: This one needs a bit more investigation. We should leave this image off the change for now. I've removed it from the PR.

File: input/assets/images/ccm/encryption-passphrase/ccm-admin-encryption-location.png
Comment (corbob) [reply input/assets/images/ccm/encryption-passphrase/ccm-admin-encryption-location.png:1]: I think this was a caching thing. It appeared to be the old one for me on the docs site until about 5 refreshes 😬 . I've backed out my change to this file.

**Review by st3phhays — APPROVED** [review]

**Pull request merged**